packages/calcite-components/src/components/block/block.tsx

```diff
@@ -416,7 +416,7 @@ export class Block
     const toggleLabel = open ? messages.collapse : messages.expand;
 
     const headerContent = (
-      <header class={CSS.header} id={IDS.header}>
+      <header class={{ [CSS.header]: true, [CSS.headerPadding]: !!heading }} id={IDS.header}>
```

Comment:
should this be `!!(heading || description)`?

```suggestion
      <header class={{ [CSS.header]: true, [CSS.headerPadding]: !!(heading || description) }} id={IDS.header}>
```